packages/common/src/rule-engine/operators/helpers.ts

```diff
@@ -1,9 +1,22 @@
-import { ConditionFn, BetweenParams, LastYearsParams, ExistsParams, Primitive } from './types';
-import { TOperation } from '../types';
+import get from 'lodash.get';
```

Comment:
Use Remeda instead of Lodash. It's considered more modern and it's still maintained.

Reply:
i havnt seen that they support install parial operations, it means that we have to install the whole dependency in our ballerine common which isnt good idea

Reply:
In the future we will add support to change the property access provider, i think we can get to it later